evaluation/analysis.py

```
@@ -10,161 +10,174 @@
-TOOL_BARPLOT_RELEVANCE = barplot_metric('tool_relevance', tool_relevance_results)
-TOOL_BARPLOT_RELEVANCE.savefig(
-    str(PATH_EVALUATION_PLOTS / f'tool_barplot_relevance.svg'),
-    format='svg'
-)
+    # get tool plots
+    tool_relevance_results: dict[str, list[float]] = {model: [] for model in set(list(tool_eval_df['model']))}
+    for _, tool_eval_row in tool_eval_df.iterrows():
+        tool_relevance_results[tool_eval_row['model']].append(tool_eval_row['tool_relevance'])
+    TOOL_BARPLOT_RELEVANCE = barplot_metric('tool_relevance', tool_relevance_results)
+    TOOL_BARPLOT_RELEVANCE.savefig(
+        str(PATH_EVALUATION_PLOTS / f'tool_barplot_relevance.svg'),
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)